packages/react-addons/package.json

```diff
@@ -2,6 +2,11 @@
   "name": "react-addons",
```

Comment:
nit: can we change this to `react-addon-template` or something? I always think it's a real thing.